### ISSUE TITLE
Ethereum Wallet V0.11.0 -  Windows version has some problems !!

### ISSUE THREAD

**Issue by huitakfung** [ethereum/mist#4006]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

- Unable to create a new account, the client program did not respond.
- The sending transaction interface is not responsive for a long time.
- Multiple regional tests have the same problem and it is recommended to fix it after testing.
@evertonfraga 
```
Version: `0.11.0`
OS & Version: windows10 64bit
Node version: `geth 1.8.11`
Number of blocks synchronized: 5973627
```

**Comment by evertonfraga** [comment]
Hi @huitakfung, that's unfortunate but true. 

We're working on a fix, please follow from here: #3988.

**Comment by huitakfung** [comment]
thanks～

许德丰
